CHANGELOG_PENDING.md

```diff
@@ -15,13 +15,13 @@
 #### Transcoder
 
 ### Bug Fixes 🐞
+- #2747 Fixed a transcoding bug that occurred when remote transcoder was removed
 
 #### CLI
 
 #### General
 
 #### Broadcaster
-- \#2709 Add logging for high keyframe interval, reduce log level for discovery loop
```

Comment:
Why is this line removed?

Reply:
This was a note from prior release that somehow was left in the pending changelog.